pvacseq/lib/main.py

```diff
@@ -193,6 +210,8 @@ def main(args_input = sys.argv[1:]):
                         type=int,
                         help="length of the peptide sequences in the input FASTA file; default 21",
                         default=21)
+    parser.add_argument('-g', '--cufflinks_genes_tracking_file', help='genes.fpkm_tracking file from Cufflinks')
+    parser.add_argument('-i', '--cufflinks_isoforms_tracking_file', help='isoforms.fpkm_tracking file from Cufflinks')
```

Comment:
I agree with removing Cufflinks from the name, if we plan on adding support for more input file types.